### ISSUE TITLE
ayu theme colors on Neovim or Vim look dim

### ISSUE THREAD

**Issue by ghost** [ayu-theme/ayu-vim#56]
Hello everyone. I really like this theme. But when I use it on neovim or vim (In terminal) under xterm or st. The colors don't even look as close as the ones on the screenshots provided by the author of this theme. I have `set termguicolors` Why do my colors look dimmed, not as bright as the ones provided in the screenshots?

**Comment by biographie** [comment]
I'm having a similar issue, have you been able to find a fix?

**Comment by ghost** [comment]
> I'm having a similar issue, have you been able to find a fix?

No, I actually gave up on this them. The colors dont even look the same on neovim TUI. I think this theme is designed for a GUI version.

**Comment by stsewd** [comment]
I'm using neovim and the theme looks great, do you have a screenshot for a better understanding  on how it looks like?
